composer.json

```
@@ -26,8 +24,8 @@
     },
     "require-dev": {
         "guzzlehttp/psr7": "^2.0",
-        "php-http/client-integration-tests": "^3.0",
-        "phpunit/phpunit": "^7.5 || ^9.4",
+        "php-http/client-integration-tests": "dev-cleanup-major",
```

adjust before merging, when client-integration-tests is released
```suggestion
        "php-http/client-integration-tests": "^4.0",
```